examples/dreambooth/train_dreambooth_flax.py

```diff
@@ -590,6 +620,39 @@ def compute_loss(params):
     logger.info(f"  Total train batch size (w. parallel & distributed) = {total_train_batch_size}")
     logger.info(f"  Total optimization steps = {args.max_train_steps}")
 
+    def checkpoint(step=None):
+        # Create the pipeline using using the trained modules and save it.
+        scheduler = FlaxPNDMScheduler(
+            beta_start=0.00085, beta_end=0.012, beta_schedule="scaled_linear", skip_prk_steps=True
+        )
```

Comment:
nit: maybe avoid hardcoding the params? Not sure about it though.

```suggestion
        scheduler, _ = FlaxPNDMScheduler.from_pretrained(
                "CompVis/stable-diffusion-v1-4", subfolder="scheduler"
        )
```